feat(radarr): simplify `SQP-1 (1080p)` and `SQP-1 (2160p)` (#1473)
TRaSH-Guides/Guides: includes/sqp/1-why.md

??? question "Why choose this quality profile? - [Click to show/hide]"

    - You want maximum compatibility between all devices and still a HQ release.

>     - You run 2 instances of Radarr and want both version or just for the 1080p ones.

Reviewer: ```suggestion
    - You run 2 instances of Radarr and want both versions or just the 1080p ones.
```

>     - You want to have minimum to no transcoding for low powered devices or remote streaming.

Reviewer: ```suggestion
    - You want to have little to no transcoding for low powered devices or remote streaming.
```

    - You want to grab streaming optimized releases.
    - Small sizes.

    ------

    BHDStudio does HQ 1080p Encodes with the following features:

    - Streaming optimized (Optimized for PLEX, emby, Jellyfin, and other streaming platforms)
    - AC3 Audio (Downmixed Lossless audio track to Dolby Digital 5.1 for optimal compatibility)
    - Small sizes
    - Good quality

>     Biggest drawback if you don't have access to the top tier indexers you won't have access to all BHDStudio releases, and you will end up with less or not streaming optimized releases.

Reviewer: ```suggestion
    Biggest drawback: If you don't have access to the top tier indexers you won't have access to all BHDStudio releases, and you will end up with less or not streaming optimized releases.
```

    - You want to have minimum till none transcoding for low powered devices or remote streaming.

>     - If available for you Streaming optimized. (Optimized for PLEX, emby, Jellyfin, and other streaming platforms)

Reviewer 2: Not sure we need this line, it's mostly repeated below?
